bin/pkgmgr

```diff
@@ -363,6 +385,12 @@ B<pkg_fmri_pattern>, publisher and release.
 Use the B<--src|dst|staging> option to specify which repositories to diff.
 Defaults to B<--src> for B<ref_repo> and to B<--dst> for B<diff_repo>.
 
+=head2 B<sign>
+
+signs the packages that match the B<pkg_fmri_pattern>, publisher and release.
+Use the B<--src|dst|staging> option to specify which repositoriy.
```

Comment:
`--src|dst|staging` seems like an sub optimal option nameing choice since it allows for calls with multiple options. I would have suggested to use `--repo={src|dst|staging}` to prevent that ambiguity by design.

Reply:
it is the same pattern which is also used for e.g. `list`. i know it's suboptimal as it allows for multiple options but was chosen as it wears the keyboard less.